### PR TITLE
feat(any-occurrences): Replace all occurrences of "any…

### PR DESCRIPTION
**Pull request by olexandor**
### Description:

Replace all occurrences of "any" in DigitalWalletProvider

### CODE REVIEW

File: packages/fscomponents/src/components/DigitalWalletProvider.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export function withDigitalWallet<P extends {}>(
 
   return class DigitalWalletProvider extends React.Component<ResultProps, WithDigitalWalletState> {
     applePay: ApplePay;
-    promises: Promise<any>[] = [];
+    promises: Promise<boolean>[] = [];
```
Comment (bweissbart) [inline packages/fscomponents/src/components/DigitalWalletProvider.tsx:47]: `initApplePay` doesn't return anything so this should be `Promise<void>`.

File: packages/fscomponents/src/components/DigitalWalletProvider.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ export function withDigitalWallet<P extends {}>(
         });
     }
 
-    async initApplePay(): Promise<any> {
+    async initApplePay(): Promise<boolean> {
```
Comment (bweissbart) [inline packages/fscomponents/src/components/DigitalWalletProvider.tsx:80]: `Promise<void>`.

**Review by bweissbart — CHANGES_REQUESTED** [review]

File: packages/fscomponents/src/components/DigitalWalletProvider.tsx (hunk 1 above)
Comment (olexandor) [reply packages/fscomponents/src/components/DigitalWalletProvider.tsx]: @bweissbart Changes request implemented.

File: packages/fscomponents/src/components/DigitalWalletProvider.tsx (hunk 2 above)
Comment (olexandor) [reply packages/fscomponents/src/components/DigitalWalletProvider.tsx]: Changes request implemented.

**Pull request closed without merging**